selfdrive/car/toyota/interface.py

```diff
@@ -163,6 +163,16 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), has_relay=False,
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.6], [0.1]]
       ret.lateralTuning.pid.kf = 0.00006
 
+    elif candidate == CAR.HIGHLANDER_TSS2:
+      stop_and_go = True
+      ret.safetyParam = 73
+      ret.wheelbase = 2.84988  # 112.2 in = 2.84988 m
+      ret.steerRatio = 16.0
+      tire_stiffness_factor = 0.8
+      ret.mass = 4700. * CV.LB_TO_KG + STD_CARGO_KG  # 4260 + 4-5 people
+      ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.18], [0.015]]  # community tuning
```

Comment:
Is this tuning correct? The values look really low compared to other TSS2 cars.

Reply:
I have tried the values for other TSS2 cars. When I drive the car on curves, it overturns and corrects all the time, while these lower values are much smoother. I did not notice any ping-ponging on straight roads either. sR 16 is good. The wheel can turn probably 30 degrees with ease.